book/01-introduction/sections/about-version-control.asc

```diff
@@ -1,62 +1,62 @@
+Metode _version control_ yang banyak dipilih oleh orang-orang adalah dengan menyalin berkas-berkas ke _directory_ lain (mungkin _directory_ yang diberi catatan waktu, jika mereka cerdas).
+Pendekatan ini sangat umum karena ini sangat sederhana, namun ini juga sangat rentan terkena galat.
+Mudah sekali untuk lupa pada _directory_ mana Anda sedang berada dan menulis ke berkas yang salah atau menyalin setiap berkas yang bukan Anda maksud secara tidak sengaja.
 
-To deal with this issue, programmers long ago developed local VCSs that had a simple database that kept all the changes to files under revision control.
+Untuk menghadapi hal ini, dahulu para _programmer_ mengembangkan VCS lokal yang memiliki _database_ sederhana yang menyimpan semua perubahan pada berkas pada _revision control_.
 
-.Local version control.
-image::images/local.png[Local version control diagram]
+.Diagram version control lokal.
+image::images/local.png[Diagram version control lokal]
 
-One of the more popular VCS tools was a system called RCS, which is still distributed with many computers today.
-Even the popular Mac OS X operating system includes the `rcs` command when you install the Developer Tools.
-RCS works by keeping patch sets (that is, the differences between files) in a special format on disk; it can then re-create what any file looked like at any point in time by adding up all the patches.
+Salah satu alat VCS yang lebih terkenal adalah sistem yang disebut dengan RCS, yang masih disebarkan dengan banyak komputer saat ini.
+Bahkan sistem operasi Mac OS X yang terkenal menyertakan perintah `rcs` ketika Anda memasang Developer Tools.
+RCS bekerja dengan cara menyimpan sekumpulan _patch_ (itulah, perbedaan antara berkas-berkas) dalam sebuah format dalam diska; itu kemudian dapat membuat ulang sebarang berkas yang terlihat sama pada satu waktu dengan menambahkan semua _patch_.
 
-==== Centralized Version Control Systems
+==== Sistem Version Control Terpusat
 
 (((version control,centralized)))
-The next major issue that people encounter is that they need to collaborate with developers on other systems.
-To deal with this problem, Centralized Version Control Systems (CVCSs) were developed.
-These systems, such as CVS, Subversion, and Perforce, have a single server that contains all the versioned files, and a number of clients that check out files from that central place. (((CVS)))(((Subversion)))(((Perforce)))
-For many years, this has been the standard for version control.
+Masalah besar selanjutnya yang dihadapi orang-orang adalah bahwa mereka butuh bekerja bersama dengan para pengembang pada sistem lain.
+Untuk menangani masalah ini, Centralized Version Control System (CVCS) dikembangkan.
+Sistem-sistem ini, seperti CVS, Subversion, dan Perforce, memiliki sebuah _server_ tunggal yang berisi semua berkas-berkas yang telah diberi versi, dan beberapa klien yang melakukan _check out_ pada berkas-berkas dari pusat tersebut.
```

Comment:
@jayvdb :+1: 

Sudah saya tambahkan lagi bagian yang terhapus. Terima kasih atas sarannya.

I have added back what have been removed. Thank you for the correction.